yarn-project/noir-contracts/src/contracts/docs_example_contract/src/main.nr

```diff
@@ -107,38 +109,32 @@ contract DocsExample {
     #[aztec(public)]
     fn lock() {
         // highlight-next-line:storage-init
-        
         storage.locked.write(true);
     }
-    // docs:end:storage-init
 
+    // docs:end:storage-init
```

Comment:
dis baaaad. me saaad

Reply:
Once again, new nargo didn't do this change.